### ISSUE TITLE
f war f ally

### ISSUE THREAD

**Issue by FSociety-Elliot** [BlockHorizons/FactionsPE#54]
hi this plugin have f war and f ally?

**Comment by Kris-Driv** [comment]
If you mean that war is alias for enemy, then yes. Otherwise no. And /f ally is implemented. All relations are implemented by the way. So you can do

- /f enemy
- /f truce
- /f neutral
- /f ally

**Comment by FSociety-Elliot** [comment]
enemy is for war?

**Comment by Kris-Driv** [comment]
Explain what /f war does.

**Comment by RubyTemple** [comment]
Is for start a war and tp in a enemy base

**Comment by Kris-Driv** [comment]
Nah, it's not implemented and I don't know If I should implement this into this plugin or maybe make another as an extension.

**Comment by RubyTemple** [comment]
Basic FactionPro has it

**Comment by Thunder33345** [comment]
not sure
every server have their own variant of "war" and it would be confusing to implement all or pick one
